homeassistant/components/homekit_controller/cover.py

```diff
@@ -216,6 +216,26 @@ def current_cover_tilt_position(self) -> int:
             tilt_position = self.service.value(
                 CharacteristicsTypes.HORIZONTAL_TILT_CURRENT
             )
+        # Recalculate to convert from arcdegree scale to percentage scale.
+        if self.is_vertical_tilt:
+            scale = 0.9
```

Comment:
0.9 is the scale to go from percent to arcdegrees. We need to go the other way and turn arcdegrees back into percent.